spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -299,8 +304,7 @@ describe('GOVUK.analyticsGA4.linkTracker', function () {
         var link = linksToTest[i]
         GOVUK.triggerEvent(link, 'click')
         expected.event_data.url = link.getAttribute('href')
-        expected.event_data.type = 'download'
-        expected.event_data.external = 'false'
```

Comment:
Why remove the check for the `external` parameter? Is it enough that `type` is set to the correct value? I notice this has been changed for the other tests as well.

Reply:
These tests will always have the same external value (the performance analyst has said our assets.publishing.service download links are external), so it has been defined at the beforeEach level once, instead of individually in each test. I think everything in this code is now external: 'true', so external: 'false' will only be useful once we implement internal link tracking. I'll make sure external is only defined in the beforeEach now.